nucleus/admin/server-mgmt/src/main/java/com/sun/enterprise/admin/servermgmt/cli/StopDomainCommand.java

```diff
@@ -170,11 +171,18 @@ protected int dasNotRunning() throws CommandException {
             ListDomainsCommand listDomains = serviceLocator.getService(ListDomainsCommand.class);
             String runningDomains = "";
             try {
-                for (String domain : listDomains.getRunningDomains()) {
-                    runningDomains = runningDomains + domain + "\n";
+                Map<String, Boolean> domains = listDomains.getDomains();
+                for (String domain : domains.keySet()) {
+                    if (domains.get(domain)) {
+                        runningDomains += "\n" + domain;
+                    }
                 }
             } catch(Exception e) {        
             }
+            runningDomains = 
+                    (runningDomains.length() < 1) 
+                    ? "\nNo domains are currently running." 
```

Comment:
I like this change, but could it be implemented using a different message name in the LocalString.properties?